src/main/java/net/sf/jsqlparser/util/cnfexpression/CNFConverter.java

```diff
@@ -19,7 +19,7 @@
  * <http://www.gnu.org/licenses/lgpl-2.1.html>.
  * #L%
  */
-package net.sf.jsqlparser.util.CNFexpression;
+package net.sf.jsqlparser.util.cnfexpression;
```

Comment:
would you mind just adding a link to https://en.wikipedia.org/wiki/Conjunctive_normal_form in your doc where you say "the CNF Form." :) other than that I really like the javadoc that you have in the trees